Remove portion of red button from showing up in "May Like" directory.

KendallDoesCoding opened the pull request:
Fixes #177

KendallDoesCoding commented:
This is fixed only thing is that the christmas trees are showing up, I want the christmas trees to show up in the bottom right/left not in the sides xD @TechStudent11 can you help on this?

TechStudent11 commented:
> This is fixed only thing is that the christmas trees are showing up, I want the christmas trees to show up in the bottom right/left not in the sides xD @TechStudent11 can you help on this?

with the current way you are putting them on screen, it's basically impossible.

KendallDoesCoding commented:
> > This is fixed only thing is that the christmas trees are showing up, I want the christmas trees to show up in the bottom right/left not in the sides xD @TechStudent11 can you help on this?
> 
> with the current way you are putting them on screen, it's basically impossible.

True, That's why I need to work on #145. For now I 25% fixed the issue but there are 4 christmas trees now.

KendallDoesCoding commented:
> > > This is fixed only thing is that the christmas trees are showing up, I want the christmas trees to show up in the bottom right/left not in the sides xD @TechStudent11 can you help on this?
> > 
> > 
> > with the current way you are putting them on screen, it's basically impossible.
> 
> True, That's why I need to work on #145. For now I 25% fixed the issue but there are 4 christmas trees now.

@TechStudent11 It's works well with #145 just looks at https://deploy-preview-145--mogulchristmas.netlify.app/you-may-like/, its coz I removed the christmas tree from the body and added it in the html, I think there's a better way to do it via a class though but for now this works as it fixes my problem. But, the issue is I only did this in the main directory and now in the you may like directory, I'm lazy to do it for the other directory's lol.

TechStudent11 commented:
> But, the issue is I only did this in the main directory and now in the you may like directory, I'm lazy to do it for the other directory's lol.

Understandable, have a great day.

KendallDoesCoding commented:
> > But, the issue is I only did this in the main directory and now in the you may like directory, I'm lazy to do it for the other directory's lol.
> 
> Understandable, have a great day.

I just did it lol and it fixed the issue in the lyrics directory lol. Have a great day!
